Tools/requirements-hypothesis.txt

```diff
@@ -1,4 +1,7 @@
 # Requirements file for hypothesis that
 # we use to run our property-based tests in CI.
 
+# see python-attrs/attrs#1326
```

Comment:
Until the root cause is clearly identified, I would prefer to refer to the CPython issue instead:

```suggestion
# see https://github.com/python/cpython/issues/122686
```